tests/cephadm/test_package_validation.py

```diff
@@ -0,0 +1,126 @@
+from cli.utilities.containers import Container
+from cli.utilities.packages import Package, PackageError, ReposError, Rpm
+from cli.utilities.utils import get_custom_repo_url
+from utility.log import Log
+
+log = Log(__name__)
+
+
+YUM_REPO_DIR = "/etc/yum.repos.d"
+
+installed_pkgs = [
+    "chrony",
+    "lvm2",
+    "podman",
+]
+
+pkgs_to_install = [
+    "cephadm-ansible",
+]
+
+rpm_dependent_pkgs = [
+    "ansible-core",
+    "ansible-collection-ansible-posix",
+    "ansible-collection-community-general",
+]
+
+rpm_client_pkgs = [
+    "ceph-common",
+    "ceph-fuse",
+    "cephfs-top",
+]
+
+
+def run(ceph_cluster, **kwargs):
```

Comment:
I would suggest to create separate methods to validate these packages with finally method having cleanup

Reply:
Updated the changes.